src/styles/_variables.scss

```diff
@@ -0,0 +1,18 @@
+// Colors
```

Comment:
Consider to name the variables without specifying a color name, so that they can be dinamyc. e.g. "primary-color", "primary-brand-color", etc.